code/game/objects/items/tanks/tank_types.dm

```diff
@@ -167,6 +167,7 @@
 /obj/item/tank/internals/emergency_oxygen/double
 	name = "double emergency oxygen tank"
 	icon_state = "emergency_double"
+	worn_icon_state = "emergency_engi"

```

Comment:
inherited from it's parent already.

Reply:
Reverted. Had it set to emergency_engi to maintain a yellow looking tank to match the icon instead of a blue one.